src/commcare_cloud/commands/deploy/utils.py

```diff
@@ -18,14 +18,15 @@ def create_release_tag(environment, repo, diff):
             print(color_error(f"Error creating release tag: {e}"))
 
 
-def announce_deploy_start(environment, service_name):
+def announce_deploy_start(environment, service_name, commcare_rev=None):
+    user = get_default_username()
+    env_name = environment.meta_config.deploy_env
+    subject = f"{user} has initiated a {service_name} deploy to {env_name}"
+    if commcare_rev:
```

Comment:
Nit: if someone runs with `--commcare-rev=master` that would cause this to be triggered, even though that's functionally equivalent to leaving this off. Would it work to use something like

```suggestion
    if commcare_rev and commcare_rev != environment.fab_settings.default_branch:
```

instead?

Reply:
Thanks for the suggestion, updated the code with the suggestions.